docs/CIRCUIT_JSON_PCB_OVERVIEW.md

```diff
@@ -344,4 +344,8 @@ export interface PcbBoard {
   center: Point
   outline?: Point[]
 }
+type PcbBoardWithOffsets = PcbBoard & {
```

Comment:
```suggestion
  outline?: Point[]
```

btw this file is supposed to be replaced with tscircuit plop (npm install -g @tscircuit/plop then run plop)